.github/scripts/sonatype-stats.scala

```diff
@@ -0,0 +1,181 @@
+package stats
+
+import java.nio.file._
+import java.time.{YearMonth, ZoneOffset}
+
+import com.softwaremill.sttp.quick._
+import upickle.default._
+import ujson.{read => _, _}
+
+object Responses {
+
+  case class UniqueIpData(total: Int)
+  implicit val uniqueIpDataRW: ReadWriter[UniqueIpData] = macroRW
+  case class UniqueIpResp(data: UniqueIpData)
+  implicit val uniqueIpRespRW: ReadWriter[UniqueIpResp] = macroRW
+
+  case class Elem(id: String, name: String)
+  implicit val elemRW: ReadWriter[Elem] = macroRW
+
+}
+
+import Responses._
+
+object Params {
+
+  // organization one was granted write access to
+  val proj = sys.env.getOrElse("SONATYPE_PROJECT", "ch.epfl.scala")
+  // actual organization used for publishing (must have proj as prefix)
+  val organization = sys.env.getOrElse("SONATYPE_PROJECT", proj)
+
+  val sonatypeUser = sys.env.getOrElse(
+    "SONATYPE_USERNAME",
+    sys.error("SONATYPE_USERNAME not set")
+  )
+  val sonatypePassword: String = sys.env.getOrElse(
+    "SONATYPE_PASSWORD",
+    sys.error("SONATYPE_PASSWORD not set")
+  )
+
+  val start = YearMonth.now(ZoneOffset.UTC)
+
+  val cutOff = start.minusMonths(4L)
+
+  // Note: this assumes the current working directory is the repository root directory!
+  val base = Paths.get("sonatype-stats")
+
+  val artifacts = Set(
+    "scalafix-core_2.12",
+    "scalafix-core_2.13"
```

Comment:
`scalafix-interfaces` would also be interesting to better understand how it is used. I guess I could add it later and run an adhoc job via a PR to backfill data for it before next month, but if you have the secret locally it might be easier to do it in this PR?